packages/rancher-kiali-server/package.yaml

```diff
@@ -1,5 +1,5 @@
-url: https://kiali.org/helm-charts/kiali-server-1.32.0.tgz
-version: 100.0.0
+url: https://kiali.org/helm-charts/kiali-server-1.35.0.tgz
+version: 100.1.0
```

Comment:
> This was fixed when Jack bumped all the versions to 100.0.0. The versions I have in this pr are a net new istio 1.10.1 and kiali 1.35.0, both now with a rancher charts version of 100.1.0.

So to clarify, a 100.0.0 already exists for 1.32.0 ? Wouldn't you need to split up the kiali package into two folders the same way we do istio ? Its possible/likely that we will wipe out whats in dev and then re-generate current dev + release-v2.6. If that happens then 100.0.0 would go away. Can we just do 1.35.0 (100.0.0+up1.35.0) for both istio 1.9 and 1.10, does it work for both ?